web/client/src/config/index.ts

```diff
@@ -2,5 +2,5 @@ export default {
   appName: "PhoneInfoga",
   appDescription:
     "Advanced information gathering & OSINT tool for phone numbers",
-  apiUrl: "/api",
+  apiUrl: "/api", // /api
```

Comment:
What's this for?

Reply:
It was juste a reminder because i was using a different route before